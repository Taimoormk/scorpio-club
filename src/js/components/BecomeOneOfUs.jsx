// ########## Import Dependencies Here ##########
import React from 'react';
import { shape, string } from 'prop-types';

// ########## Import Containers Here ##########

// ########## Import Components Here ##########

const BecomeOneOfUs = (props) => {
  const { becomeOneOfUs } = props;
  return (
    <section id="become-one-of-us">
      <div className="wrapper">
        <h3 className="become-one-of-us-heading">{becomeOneOfUs.sectionHeading}</h3>
        <h4 className="become-one-of-us-subheading">{becomeOneOfUs.sectionH3}</h4>
        <button className="become-one-of-us-button">{becomeOneOfUs.sectionButtonText}</button>
        <div className="filler" />
      </div>
    </section>
  );
}

BecomeOneOfUs.propTypes = {
  becomeOneOfUs: shape({
    sectionHeading: string.isRequired,
    sectionH3: string.isRequired,
    sectionButtonText: string.isRequired,
  }).isRequired,
}

// function mapStateToProps(state) {
//   return {
//     key: null
//   };
// };

export default BecomeOneOfUs;