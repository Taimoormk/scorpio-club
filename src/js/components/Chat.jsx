// ########## Import Dependencies Here ##########
import React, { Component } from 'react';
import { arrayOf, shape, func, bool } from 'prop-types';
import { connect } from 'react-redux';
import ChatBot from 'react-simple-chatbot';

// ########## Import Containers Here ##########

// ########## Import Components Here ##########

export default class Chat extends Component {
  render() {
    return (
      <div id="chat">
        <ChatBot
          steps={[
            {
              id: 'hello-world',
              message: 'Hello World!',
              end: true,
            },
          ]}
        />
      </div>
    )
  }
}