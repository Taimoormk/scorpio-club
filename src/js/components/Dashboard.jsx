// ########## Import Dependencies Here ##########
import React, { Component } from 'react';
// import { arrayOf, shape, func, bool } from 'prop-types';
// import { connect } from 'react-redux';
import {
  Route,
  Switch,
} from 'react-router-dom';
import { Scrollbars } from 'react-custom-scrollbars';

// ########## Import Containers Here ##########

// ########## Import Components Here ##########
import DashboardAppBar from './Dashboard/DashboardAppBar';
import SideFeed from './Dashboard/SideFeed';
import Profile from './Dashboard/Profile';
import CreateGifs from './Dashboard/CreateGifs';
import NotableScorpios from './Dashboard/NotableScorpios';
import FromOurScorpios from './Dashboard/FromOurScorpios';
import ShareScorpioTraits from './Dashboard/ShareScorpioTraits';
import Marketplace from './Dashboard/Marketplace';
import Settings from './Dashboard/Settings';
import Ad from './Dashboard/Ad';
import DashboardHome from './Dashboard/DashboardHome';
import DashboardMenuData from './Dashboard/DashboardMenuData';

export default class Dashboard extends Component {

  state = { visible: false };

  toggleVisibility = () => this.setState({ visible: !this.state.visible });

  render() {
    const { visible } = this.state;
    return (
      <div id="dashboard">
        <DashboardAppBar
          toggleVisibility={this.toggleVisibility}
        />
        <div className="dashboard-content">
          <div className="dashboard-main-content">
            <Scrollbars>
              <Switch>
                <Route
                  path="/dashboard/profile"
                  render={
                    function (routeProps) {
                      return (
                        <Profile
                          visible={visible}
                          breadcrumbs='Profile'
                          // toggleVisibility={toggleVisibility}
                          {...routeProps}
                        />
                      )
                    }
                  }
                />
                <Route
                  path="/dashboard/create-gifs"
                  render={
                    function (routeProps) {
                      return (
                        <CreateGifs
                          visbible={visible}
                          breadcrumbs='Create Gifs'
                          {...routeProps}
                        />
                      )
                    }
                  }
                />
                <Route
                  path="/dashboard/notable-scorpios"
                  render={
                    function (routeProps) {
                      return (
                        <NotableScorpios
                          visible={visible}
                          breadcrumbs='Notable Scorpios'
                          {...routeProps}
                        />
                      )
                    }
                  }
                />
                <Route
                  path="/dashboard/from-our-scorpios"
                  render={
                    function (routeProps) {
                      return (
                        <FromOurScorpios
                          visibile={visible}
                          breadcrumbs='From Our Scorpios'
                          {...routeProps}
                        />
                      )
                    }
                  }
                />
                <Route
                  path="/dashboard/share-scorpio-traits"
                  render={
                    function (routeProps) {
                      return (
                        <ShareScorpioTraits
                          visibile={visible}
                          breadcrumbs='Share Scorpio Traits'
                          {...routeProps}
                        />
                      )
                    }
                  }
                />
                <Route
                  path="/dashboard/marketplace"
                  render={
                    function (routeProps) {
                      return (
                        <Marketplace
                          visible={visible}
                          breadcrumbs='Marketplace'
                          {...routeProps}
                        />
                      )
                    }
                  }
                />
                <Route
                  path="/dashboard/settings"
                  render={
                    function (routeProps) {
                      return (
                        <Settings
                          visibile={visible}
                          breadcrumbs='Settings'
                          {...routeProps}
                        />
                      )
                    }
                  }
                />
                {/* <Route path="/dashboard/create-gifs" component={CreateGifs} /> */}
                {/* <Route path="/dashboard/notable-scorpios" component={NotableScorpios} /> */}
                {/* <Route path="/dashboard/from-our-scorpios" component={FromOurScorpios} /> */}
                {/* <Route path="/dashboard/share-scorpio-traits" component={ShareScorpioTraits} /> */}
                {/* <Route path="/dashboard/marketplace" component={Marketplace} /> */}
                {/* <Route path="/dashboard/settings" component={Settings} /> */}
                <Route
                  path="/dashboard"
                  toggleVisibility={this.toggleVisibility}
                  render={
                    function (routeProps, props) {
                      // console.log('Dashboard', props);
                      // const { toggleVisibility } = props;
                      return (
                        <DashboardHome
                          visible={visible}
                          breadcrumbs='Dashboard'
                          // toggleVisibility={toggleVisibility}
                          {...routeProps}
                        />
                      )
                    }
                  }
                />
              </Switch>
            </Scrollbars>
          </div>
          <div className="dashboard-side-feed">
            <Scrollbars>
              <SideFeed />
              <Ad />
            </Scrollbars>
          </div>
        </div>
      </div>
    );
  }
}