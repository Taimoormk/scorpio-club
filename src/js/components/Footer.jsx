// ########## Import Dependencies Here ##########
import React, { Component } from 'react';
// import PropTypes from 'prop-types';
import { connect } from 'react-redux'

// ########## Import Containers Here ##########

// ########## Import Components Here ##########

class Footer extends Component {

  componentDidMount() {

  }

  render() {
    return (
      <section id="footer">
        <div className="wrapper">
          <div className="footer-container-left">
            <p className="footer-heading">Scorpio Club</p>
            <p className="footer-disclaimer">Â© 2018 Scorpio Club | All rights reserved.</p>
          </div>
          <div className="footer-container-right">
            <p className="footer-heading" />
            <div className="footer-disclaimer">
              <a href="https://facebook.com" target="_blank" rel="noopener noreferrer"><i className="fab fa-facebook fa-2x" /></a>
              <a href="https://twitter.com" target="_blank" rel="noopener noreferrer"><i className="fab fa-twitter-square fa-2x" /></a>
              <a href="https://instagram.com" target="_blank" rel="noopener noreferrer"><i className="fab fa-instagram fa-2x" /></a>
            </div>
          </div>
          <div className="clear" />
        </div>
      </section>
    );
  }
}

Footer.propTypes = {
}

// function mapStateToProps(state) {
//   return {
//     key: null
//   };
// };

export default connect(null, {})(Footer);