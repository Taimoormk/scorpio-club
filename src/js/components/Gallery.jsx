// ########## Import Dependencies Here ##########
import React, { Component } from 'react';
import { arrayOf, shape, string } from 'prop-types';

// ########## Import Containers Here ##########

// ########## Import Components Here ##########

class Gallery extends Component {

  static propTypes = {
    sectionTitle: string.isRequired,
    fromOurScorpiosData: arrayOf(
      shape({
        postImg: string,
      })
    ),
    someNotableScorpiosData: arrayOf(
      shape({
        postImg: string,
      })
    ),
  }

  static defaultProps = {
    fromOurScorpiosData: null,
    someNotableScorpiosData: null
  }

  fromOurScorpiosRender() {
    const { fromOurScorpiosData } = this.props;
    return (
      fromOurScorpiosData.map((image, k) => {
        const postImg = fromOurScorpiosData[k].postImg;
        const postCategoty = fromOurScorpiosData[k].postCategory;
        const postDate = fromOurScorpiosData[k].postDate;
        return (
          <div
            key={k}
            className="from-our-scorpios-gallery"
          >
            <div className="from-our-scorpio-gallery-item">
              <div className="from-our-scorpio-gallery-item-overlay" />
              <img
                src={postImg}
                className="from-our-scorpios-gallery-image"
                alt="from our scorpios gallery item"
              />
              <div className="from-our-scorpios-gallery-overlay-content">
                <h4 className="from-our-scorpios-gallery-overlay-category">{postCategoty}</h4>
                <p className="from-our-scorpios-gallery-overlay-date">{postDate}</p>
              </div>
            </div>
          </div>
        )
      })
    )
  }

  someNotableScorpiosRender() {
    const { someNotableScorpiosData } = this.props;
    return (
      someNotableScorpiosData.map((image, k) => {
        const postImg = someNotableScorpiosData[k].postImg;
        const postCategory = someNotableScorpiosData[k].postCategory;
        const postDate = someNotableScorpiosData[k].postDate;
        return (
          <div
            key={k}
            className="some-notable-scorpios-gallery"
          >
            <div className="some-notable-scorpios-gallery-item">
              <div className="some-notable-scorpios-gallery-item-overlay" />
              <img
                src={postImg}
                className="some-notable-scorpios-gallery-item"
                alt="some notable scorpios gallery item"
              />
              <div className="some-notable-scorpios-gallery-overlay-content">
                <h4 className="some-notable-scorpios-gallery-overlay-category">{postCategory}</h4>
                <p className="some-notable-scorpios-gallery-overlay-date">{postDate}</p>
              </div>
            </div>
          </div>
        )
      })
    )
  }

  render() {
    const { sectionTitle } = this.props;
    return (
      <div id="gallery">
        {
          sectionTitle === "From Our Scorpios" && this.fromOurScorpiosRender()
          ||
          sectionTitle === "Some Notable Scorpios" && this.someNotableScorpiosRender()
        }
      </div>
    );
  }
}

export default Gallery;