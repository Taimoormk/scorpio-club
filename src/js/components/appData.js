const appData = [
  {
    id: Math.random(),
    sectionTitle: 'Scorpio Club',
    sectionHeading: 'Scorpio Club',
    sectionLine1: 'Scorpio is the best sign ever!',
    sectionLine2: 'Join us to know all the reasons WHY!',
    sectionButtonText: 'let me in'
  },
  {
    id: Math.random(),
    promoPosts: [
      {
        id: 1,
        postTitle: 'The Problem Solver1',
        postImage: './images/scorpio-club-promo-1.png',
        postDescription: 'In a fix or deep trouble? As a Scorpio for help! Yes, Scorpio have an uncanny ability to work out solutions from scratch.',
        imageDirection: 'left',
      },
      {
        id: 2,
        postTitle: 'The Problem Solver2',
        postImage: './images/scorpio-club-promo-2.png',
        postDescription: 'In a fix or deep trouble? As a Scorpio for help! Yes, Scorpio have an uncanny ability to work out solutions from scratch.',
        imageDirection: 'right',
      },
      {
        id: 3,
        postTitle: 'The Problem Solver3',
        postImage: './images/scorpio-club-promo-3.png',
        postDescription: 'In a fix or deep trouble? As a Scorpio for help! Yes, Scorpio have an uncanny ability to work out solutions from scratch.',
        imageDirection: 'left',
      },
      {
        id: 4,
        postTitle: 'The Problem Solver4',
        postImage: './images/scorpio-club-promo-4.png',
        postDescription: 'In a fix or deep trouble? As a Scorpio for help! Yes, Scorpio have an uncanny ability to work out solutions from scratch.',
        imageDirection: 'right',
      }
    ]
  },
  {
    id: Math.random(),
    sectionTitle: 'From Our Scorpios',
    sectionHeading: 'From Our Scorpios',
    sectionLine1: 'With our drag and drop system you can create pages in minutes!',
    sectionLine2: 'Another line lets see how it looks like',
    sectionButtonText: 'see more',
    fromOurScorpiosData: [
      {
        id: 1,
        postTitle: 'Best Scorpios',
        postImg: './images/bill-gates.jpg',
        postCategory: 'Web',
        postDate: '16 Nov, 2017'
      },
      {
        id: 2,
        postTitle: 'Best Scorpios',
        postImg: './images/bill-gates.jpg',
        postCategory: 'Web',
        postDate: '16 Nov, 2017'
      },
      {
        id: 3,
        postTitle: 'Best Scorpios',
        postImg: './images/bill-gates.jpg',
        postCategory: 'Web',
        postDate: '16 Nov, 2017'
      },
      {
        id: 4,
        postTitle: 'Best Scorpios',
        postImg: './images/bill-gates.jpg',
        postCategory: 'Web',
        postDate: '16 Nov, 2017'
      },
      {
        id: 5,
        postTitle: 'Best Scorpios',
        postImg: './images/bill-gates.jpg',
        postCategory: 'Web',
        postDate: '16 Nov, 2017'
      },
      {
        id: 6,
        postTitle: 'Best Scorpios',
        postImg: './images/bill-gates.jpg',
        postCategory: 'Web',
        postDate: '16 Nov, 2017'
      }
    ]
  },
  {
    id: Math.random(),
    sectionTitle: 'Some Notable Scorpios',
    sectionHeading: 'Some Notable Scorpios',
    sectionLine1: 'With our drag and drop system you can create pages in minutes!',
    sectionLine2: 'Another line lets see how it looks like',
    sectionButtonText: 'see more',
    someNotableScorpiosData: [
      {
        id: 1,
        postTitle: 'Best Scorpios',
        postImg: './images/bill-gates.jpg',
        postCategory: 'Web',
        postDate: '16 Nov, 2017'
      },
      {
        id: 2,
        postTitle: 'Best Scorpios',
        postImg: './images/bill-gates.jpg',
        postCategory: 'Web',
        postDate: '16 Nov, 2017'
      },
      {
        id: 3,
        postTitle: 'Best Scorpios',
        postImg: './images/bill-gates.jpg',
        postCategory: 'Web',
        postDate: '16 Nov, 2017'
      },
      {
        id: 4,
        postTitle: 'Best Scorpios',
        postImg: './images/bill-gates.jpg',
        postCategory: 'Web',
        postDate: '16 Nov, 2017'
      },
      {
        id: 5,
        postTitle: 'Best Scorpios',
        postImg: './images/bill-gates.jpg',
        postCategory: 'Web',
        postDate: '16 Nov, 2017'
      },
      {
        id: 6,
        postTitle: 'Best Scorpios',
        postImg: './images/bill-gates.jpg',
        postCategory: 'Web',
        postDate: '16 Nov, 2017'
      }
    ]
  },
  {
    id: Math.random(),
    sectionTitle: 'Become One Of Us',
    sectionHeading: 'Become One Of Us',
    sectionLine1: 'Society excited by cottage private an it esteems. Fully begin on by wound an.',
    sectionLine2: 'Girl rich in do up or both. At declared in as rejoiced of together.',
    sectionButtonText: 'see more'
  },
  {
    id: Math.random(),
    sectionTitle: 'Lets be in Touch',
    sectionHeading: 'Lets be in Touch',
    sectionLine1: 'Society excited by cottage private an it esteems. Fully begin on by wound an.',
    sectionLine2: 'Girl rich in do up or both. At declared in as rejoiced of together.',
    sectionButtonText: 'submit'
  },
  {
    id: Math.random(),
    sectionTitle: 'Footer',
    sectionHeading: 'Scorpio Club',
    sectionH3: '',
    sectionH4: '',
    sectionPara: 'Â© 2018 Scorpio Club | All rights reserved.',
    sectionButtonText: 'submit'
  }
]

export default appData;