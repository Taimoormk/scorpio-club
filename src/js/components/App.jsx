// ########## Import Dependencies Here ##########
import React, { Component } from 'react';
import { arrayOf, shape, func, bool } from 'prop-types';
import { connect } from 'react-redux';
import {
  BrowserRouter as Router,
  Link,
  Route,
  Switch
} from 'react-router-dom';

// ########## Import Containers Here ##########

// ########## Import Components Here ##########
import Home from './Home';
import Dashboard from './Dashboard';
import * as actions from '../actions';

export class App extends Component {

  constructor(props) {
    super(props);
    const { loadAppAction } = this.props;
    loadAppAction();
  }

  render() {
    return (
      <Router>
        <Switch>
          <Route path="/dashboard" component={Dashboard} />
          <Route path="/" component={Home} />
        </Switch>
      </Router>
    );
  }
}

export default connect(
  null, {
    loadAppAction: actions.loadAppAction,
  })
  (App);