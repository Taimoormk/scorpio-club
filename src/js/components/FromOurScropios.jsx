// ########## Import Dependencies Here ##########
import React from 'react';
import { shape, string } from 'prop-types';
import Typed from 'react-typed';

// ########## Import Containers Here ##########

// ########## Import Components Here ##########
import Gallery from './Gallery';

const FromOurScorpios = (props) => {
  const { fromOurScorpios } = props;
  return (
    <section id="from-our-scorpios">
      <div className="wrapper">
        <h3 className="from-our-scorpios-heading">{fromOurScorpios.sectionHeading}</h3>
        {/* <h4 className="from-our-scorpios-subheading">{fromOurScorpios.sectionH3}</h4> */}
        <Typed
          className="from-our-scorpios-subheading"
          strings={[fromOurScorpios.sectionLine1, fromOurScorpios.sectionLine2]}
          typeSpeed={40}
          backSpeed={50}
          loop
        />
        <div className="from-our-scorpios-collection-container">
          <Gallery
            sectionTitle={fromOurScorpios.sectionTitle}
            fromOurScorpiosData={fromOurScorpios.fromOurScorpiosData}
          />
          <button className="from-our-scorpios-button">{fromOurScorpios.sectionButtonText}</button>
          <div className="filler" />
        </div>
      </div>
    </section>
  );
}

FromOurScorpios.propTypes = {
  fromOurScorpios: shape({
    sectionHeading: string.isRequired,
    sectionLine1: string.isRequired,
    sectionLine2: string.isRequired,
    sectionButtonText: string.isRequired,
  }).isRequired,
}

export default FromOurScorpios;