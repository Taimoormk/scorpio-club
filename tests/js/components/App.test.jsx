// ########## Import Dependencies Here ##########
import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

// ########## Import Components Here ##########
import { App } from '../../../src/js/components/App';

Enzyme.configure({ adapter: new Adapter() });

describe('App Component', () => {
  let initialLoadData,
    activateModalAction,
    deactivateModalAction,
    toggleModal,
    heroData,
    promoData,
    fromOurScorpios,
    someNotableScorpios,
    becomeOneOfUs,
    letsBeInTouch,
    footer,
    wrapper;

  beforeEach(() => {
    initialLoadData = [
      { sectionHeading: "Scorpio Club", sectionLine1: "Scorpio is the best sign ever!", sectionLine2: "Join us to know all the reasons WHY!", sectionButtonText: "let me in" }, 
      { promoPosts: [ { postTitle: "The Problem Solver1", postImage: "./images/scorpio-club-promo-1.png", postDescription: "In a fix or deep trouble? As a Scorpio for help! Yes, Scorpio have an uncanny ability to work out solutions from scratch." }, { postTitle: "The Problem Solver1", postImage: "./images/scorpio-club-promo-1.png", postDescription: "In a fix or deep trouble? As a Scorpio for help! Yes, Scorpio have an uncanny ability to work out solutions from scratch." }, { postTitle: "The Problem Solver1", postImage: "./images/scorpio-club-promo-1.png", postDescription: "In a fix or deep trouble? As a Scorpio for help! Yes, Scorpio have an uncanny ability to work out solutions from scratch." }, { postTitle: "The Problem Solver1", postImage: "./images/scorpio-club-promo-1.png", postDescription: "In a fix or deep trouble? As a Scorpio for help! Yes, Scorpio have an uncanny ability to work out solutions from scratch." } ] }, 
      { sectionHeading: "From Our Scorpios", sectionLine1: "With our drag and drop system you can create pages in minutes!", sectionLine2: "Another line lets see how it looks like", sectionButtonText: "see more" }, 
      { sectionHeading: "Some Notable Scorpios", sectionLine1: "With our drag and drop system you can create pages in minutes!", sectionLine2: "Another line lets see how it looks like", sectionButtonText: "see more" }, 
      { sectionHeading: "Become One Of Us", sectionLine1: "Society excited by cottage private an it esteems. Fully begin on by wound an.", sectionLine2: "Girl rich in do up or both. At declared in as rejoiced of together.", sectionButtonText: "see more" }, 
      { sectionHeading: "Lets be in Touch", sectionLine1: "Society excited by cottage private an it esteems. Fully begin on by wound an.", sectionLine2: "Girl rich in do up or both. At declared in as rejoiced of together.", sectionButtonText: "see more" }, 
      { sectionHeading: "Scorpio Club", sectionPara: "Â© 2018 Scorpio Club | All rights reserved." }
    ];
    activateModalAction = jest.fn();
    deactivateModalAction = jest.fn();
    toggleModal = { modalActive: false };
    heroData = initialLoadData[0];
    promoData = initialLoadData[1].promoPosts[0];
    fromOurScorpios = initialLoadData[2];
    someNotableScorpios = initialLoadData[3];
    becomeOneOfUs = initialLoadData[4];
    letsBeInTouch = initialLoadData[5];
    footer = initialLoadData[6];
    wrapper = shallow(
      <App
        initialLoadData={initialLoadData}
        activateModalAction={activateModalAction}
        deactivateModalAction={deactivateModalAction}
        toggleModal={toggleModal}
        promoData={promoData}
        fromOurScorpios={fromOurScorpios}
        someNotableScorpios={someNotableScorpios}
        becomeOneOfUs={becomeOneOfUs}
        letsBeInTouch={letsBeInTouch}
        footer={footer}
      />
    );
  });

  it('renders', () => {
    expect(wrapper).toMatchSnapshot();
  });
  it('passes on activateModalAction', () => {
    expect(activateModalAction.mock.calls.length).toBe(0);
  });
  it('passes on deactivateModalAction', () => {
    expect(deactivateModalAction.mock.calls.length).toBe(0);
  });
});