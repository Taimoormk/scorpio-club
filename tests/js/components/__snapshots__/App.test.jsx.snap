// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Component renders 1`] = `
<div
  className="app"
>
  <Hero
    activateModalAction={[MockFunction]}
    heroData={
      Object {
        "sectionButtonText": "let me in",
        "sectionHeading": "Scorpio Club",
        "sectionLine1": "Scorpio is the best sign ever!",
        "sectionLine2": "Join us to know all the reasons WHY!",
      }
    }
  />
  <div
    className="promo-gradient"
  >
    <Promo
      promoData={
        Object {
          "postDescription": "In a fix or deep trouble? As a Scorpio for help! Yes, Scorpio have an uncanny ability to work out solutions from scratch.",
          "postImage": "./images/scorpio-club-promo-1.png",
          "postTitle": "The Problem Solver1",
        }
      }
    />
    <Promo
      promoData={
        Object {
          "postDescription": "In a fix or deep trouble? As a Scorpio for help! Yes, Scorpio have an uncanny ability to work out solutions from scratch.",
          "postImage": "./images/scorpio-club-promo-1.png",
          "postTitle": "The Problem Solver1",
        }
      }
    />
    <Promo
      promoData={
        Object {
          "postDescription": "In a fix or deep trouble? As a Scorpio for help! Yes, Scorpio have an uncanny ability to work out solutions from scratch.",
          "postImage": "./images/scorpio-club-promo-1.png",
          "postTitle": "The Problem Solver1",
        }
      }
    />
    <Promo
      promoData={
        Object {
          "postDescription": "In a fix or deep trouble? As a Scorpio for help! Yes, Scorpio have an uncanny ability to work out solutions from scratch.",
          "postImage": "./images/scorpio-club-promo-1.png",
          "postTitle": "The Problem Solver1",
        }
      }
    />
  </div>
  <FromOurScorpios
    activateModalAction={[MockFunction]}
    fromOurScorpios={
      Object {
        "sectionButtonText": "see more",
        "sectionHeading": "From Our Scorpios",
        "sectionLine1": "With our drag and drop system you can create pages in minutes!",
        "sectionLine2": "Another line lets see how it looks like",
      }
    }
  />
  <SomeNotableScorpios
    activateModalAction={[MockFunction]}
    someNotableScorpios={
      Object {
        "sectionButtonText": "see more",
        "sectionHeading": "Some Notable Scorpios",
        "sectionLine1": "With our drag and drop system you can create pages in minutes!",
        "sectionLine2": "Another line lets see how it looks like",
      }
    }
  />
  <BecomeOneOfUs
    becomeOneOfUs={
      Object {
        "sectionButtonText": "see more",
        "sectionHeading": "Become One Of Us",
        "sectionLine1": "Society excited by cottage private an it esteems. Fully begin on by wound an.",
        "sectionLine2": "Girl rich in do up or both. At declared in as rejoiced of together.",
      }
    }
  />
  <LetsBeInTouch
    letsBeInTouch={
      Object {
        "sectionButtonText": "see more",
        "sectionHeading": "Lets be in Touch",
        "sectionLine1": "Society excited by cottage private an it esteems. Fully begin on by wound an.",
        "sectionLine2": "Girl rich in do up or both. At declared in as rejoiced of together.",
      }
    }
  />
  <Footer
    footer={
      Object {
        "sectionHeading": "Scorpio Club",
        "sectionPara": "Â© 2018 Scorpio Club | All rights reserved.",
      }
    }
  />
</div>
`;
